HTML:CSS learning project/style.css

```diff
@@ -1,3 +1,31 @@
-h1 {
-  color: red;
+header{
+  color:white;
+  width: 100%;
+  position: fixed;        
+  background-color:black;
+  padding: 10px 0;
+  top: 0;
+}
+
+footer {
+  color:white;
+  width: 100%;
+  position: fixed;        
+  background-color:black;
+  padding: 10px 0;
+  bottom: 0;
+}
+nav a{
+  padding: 7px 25px;
+  display: inline-block;
+}
```

Comment:
Also, try to keep CSS as clean as possible. Try to leave one empty line after CSS bracket closure. It isn't essential for now, however, keeping good formatting from day 1 could help in the long-run.